resources/mirror-archive

```diff
@@ -4,20 +4,35 @@
 #
 
 BASEDIR="$(realpath $(dirname $0)/..)"
+SCRIPTNAME="$(basename $0)"
 
+CONFIG_FILE="$BASEDIR/config"
 REPO_DIR="$BASEDIR/static/ubuntu"
+
 # Variables used by reprepro
 export REPREPRO_BASE_DIR="$BASEDIR/reprepro"
 export REPREPRO_CONFIG_DIR="$REPREPRO_BASE_DIR/conf"
 export GNUPGHOME="$REPREPRO_BASE_DIR/.gnupg"
 
+
+log() {
+    logger -t "$SCRIPTNAME" -i "$@"
+}
+
 # Load the config file if present.
 #
 # currently, it must define the $SUITE variable
 #
-. "$BASEDIR/config" || exit
+if [ -f "$CONFIG_FILE" ]; then
+    . "$CONFIG_FILE"
+else
+    log "config file not found, exiting"
+    exit
```

Comment:
Actually i didn't want to make the cron job fail because of this.
The config file is removed if the mirror-uri is unset, which is a way of disabiling mirroring.